src/types/policyData.go

```diff
@@ -309,3 +309,15 @@ type KubeArmorPolicy struct {
 
 	Spec KnoxSystemSpec `json:"spec,omitempty" yaml:"spec,omitempty"`
 }
+
+// =============== //
+// == Policy DB == //
+// =============== //
+type Policy struct {
+	Type        string
```

Comment:
IMO, we should add another field - `Kind`. Even though the `Type` specifies whether it is `system` or `network`. Within `system` policies, we have `KubeArmorPolicy` and `KubeArmorHostPolicy`. Likewise of the network - `CNP` and `CCNP`. So having `Kind` here will help us properly parse the contents of `PolicyYaml`.